docs/cmdline-opts/retry.md

```diff
@@ -23,7 +23,7 @@ response code.
 
 When curl is about to retry a transfer, it first waits one second and then for
 all forthcoming retries it doubles the waiting time until it reaches 10
-minutes which then remains delay between the rest of the retries. By using
+minutes which then remains as the delay between the rest of the retries. By using
```

Comment:
How about being a little more explicit and say:

 "which then remains the set fixed delay time between the rest of the retries." ?

Reply:
Looks good, did it in https://github.com/curl/curl/pull/17443/commits/71cd23a1d8813dada3aec05dc006a96c56f0fd25